src/test/scala/org/broadinstitute/dsde/rawls/graph/OrientDbBasicTest.scala

```diff
@@ -0,0 +1,62 @@
+package org.broadinstitute.dsde.rawls.graph
```

Comment:
Not really. It's just to illustrate how OrientDB / the test fixture works. Perhaps it would make more sense to put in a Gist / Confluence.